public/css/lrr.css

```diff
@@ -638,11 +636,9 @@ body.swal2-shown>[aria-hidden="true"] {
   }
 
   #i2,
-  #customheader1,
-  #customheader2,
+  [id^="customheader"],
```

Comment:
These changes seem to only collapse the headers, not the columns.  
![image](https://github.com/user-attachments/assets/946eceec-167b-43a6-bea6-3a6089910edc)